From 0c8c8795f08abb95161c610a0a6dff22d45742d4 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Wed, 30 Apr 2025 11:49:45 -0500
Subject: [PATCH] server-test: Fix use of g_test_bug()

---
 tests/server-test.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/server-test.c b/tests/server-test.c
index 8d0c38bb0..f5953b6ca 100644
--- a/tests/server-test.c
+++ b/tests/server-test.c
@@ -255,7 +255,7 @@ do_invalid_percent_encoding_paths_test (ServerData *sd, gconstpointer test_data)
 	SoupMessage *msg;
 	GUri *uri;
 
-	g_test_bug ("262");
+	g_test_bug ("https://gitlab.gnome.org/GNOME/libsoup/-/issues/262");
 
 	session = soup_test_session_new (NULL);
 
-- 
GitLab

